dashboard/app/views/ops_mailer/workshop_reminder.html.haml

```diff
@@ -1,7 +1,7 @@
 %p
   Dear #{@recipient.properties['ops_first_name']},
```

Comment:
I think you have access to the method `@recipient.ops_first_name` so you don't have to go through properties